integration/test/draining/main_test.go

```diff
@@ -3,35 +3,40 @@
 package draining
 
 import (
+	"context"
+	"fmt"
+	"os"
 	"testing"
 
 	"github.com/giantswarm/e2e-harness/pkg/framework"
 	"github.com/giantswarm/e2eclients/aws"
+	e2esetup "github.com/giantswarm/e2esetup/aws"
+	"github.com/giantswarm/e2esetup/aws/env"
```

Comment:
Changes are bigger for the draining test because it hasn't been extracted to `e2etests`.